### PR TITLE
Fix agfs 10 conferences

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

File: fee_calculator/apps/calculator/management/commands/generatefees.py [hunk 1]
```diff
@@ -407,21 +412,50 @@ def generate_agfs10_fees(agfs_scheme, basic_fees_path, misc_fees_path):
...
+
+                        if fee['DEFENDANT_UPLIFT_ALLOWED'] == 'Y':
+                            price.modifiers.add(defendant_modifier)
+                        if fee['CASE_UPLIFT_ALLOWED'] == 'Y':
+                            price.modifiers.add(case_modifier)
+
+                        price.save()
+                else:
+                    if fee['BIST_BILL_SUB_TYPE'] in ['AGFS_COMMITTAL', 'AGFS_CONTEMPT']:
+                        # correct erroneous 'HALFDAY' unit for these fees
```
Comment (jsugarman) [inline fee_calculator/apps/calculator/management/commands/generatefees.py]: corrected now?

Comment (ibrechin) [reply fee_calculator/apps/calculator/management/commands/generatefees.py]: It was corrected in the actual prices in a previous PR (https://github.com/ministryofjustice/laa-fee-calculator/pull/50), but I thought I should add it to the generation code as well.

File: fee_calculator/apps/calculator/fixtures/price.json [hunk 2]
```diff
@@ -1357904,5 +1357976,1445 @@
...
+  "fields": {
+    "scenario": 8,
+    "scheme": 3,
+    "advocate_type": "JUNIOR",
+    "offence_class": null,
+    "fee_type": 13,
+    "unit": "HOUR",
+    "fixed_fee": "0.00000",
+    "fee_per_unit": "40.00000",
+    "limit_from": 9,
```
Comment (jsugarman) [inline fee_calculator/apps/calculator/fixtures/price.json]: From looking at the [applicable regs](http://www.legislation.gov.uk/uksi/2013/435/schedule/1/paragraph/19/made), and without fully understanding the code itself, are the `limit_from` and `limit_too` representing the following:  7 to 8 (1 extra CV), 9 to 12 (two extra), 13 to 18 (three extra)?

Comment (jsugarman) [reply fee_calculator/apps/calculator/fixtures/price.json]: as discussed 👍

**Pull request merged**